source/ScreensaverHero.hx

```diff
@@ -0,0 +1,92 @@
+package;
+
+import flixel.FlxG;
+import flixel.FlxSprite;
+import flixel.math.FlxRandom;
+import flixel.util.FlxSpriteUtil;
+
+class ScreensaverHero extends FlxSprite
+{
+	public var direction:Int;
+
+	public function new(x:Int, y:Int)
+	{
+		super(x, y);
+		var randomDirection:FlxRandom = new FlxRandom();
```

Comment:
We'll be talking about this in class later, but you can also save your program the overhead of setting up a new random number generator and instead use [the one that's exposed as a static field of `FlxG` - `FlxG.random`](https://api.haxeflixel.com/flixel/FlxG.html#random) 

What you've got definitely works, this would just be a way to cut out some unnecessary setup.

Quick aside, I think the naming of this variable is a bit confusing - an FlxRandom object is a random number generator, but the name `randomDirection` makes me think it's the value that you want to assign to your `direction` field.